app/assets/javascripts/components/reorderable-list.js

```diff
@@ -85,5 +90,19 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  ReorderableList.prototype.triggerEvent = function (element, eventName, detail) {
```

Comment:
Minor: we don't use the `detail` parameter, so we could lose it, I think.

Reply:
Removed the detail – not used in the current scenario, indeed